lib/grape/validations/params_scope.rb

```diff
@@ -464,6 +466,13 @@ def derive_validator_options(validations)
           fail_fast:   validations.delete(:fail_fast) || false
         }
       end
+
+      def validates_presence(validations, attrs, doc_attrs, opts)
+        return unless validations.key?(:presence) && validations[:presence]
+        validate(:presence, validations[:presence], attrs, doc_attrs, opts)
+        validations.delete(:presence)
```

Comment:
Same as above, don't alter validates, make a copy. The caller may be removing things if needed.